src/demo-app/demo-app/demo-app.scss

```diff
@@ -1,3 +1,19 @@
+// The default indigo-pink theme will load through the index file. This makes the SCSS build faster
```

Comment:
Can we actually pull this out into a custom theme file and lose the prebuilt altogether? I see your point with saving build time, but dropping in here makes the theme code harder to find. This is supposed to be an example, so it might confuse people as is.